src/main/java/com/codegym/shopyy/model/CartItem.java

```diff
@@ -0,0 +1,39 @@
+package com.codegym.shopyy.model;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.Min;
+import lombok.*;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Table(name = "cart_items")
+@JsonIgnoreProperties({"cart", "product"})
+public class CartItem {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+//    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
remove comments if not using or use // TODO: what you will do in the future

Reply:
to do: 
the commented-out code sections are intended to establish relationships with other entities in the future

Reply:
resoled